README.rst

```diff
@@ -9,9 +9,13 @@ Cite us:
 
 Status master branch (stable):
 
-.. image:: https://travis-ci.org/DIRACGrid/DIRAC.svg?branch=master
-   :target: https://travis-ci.org/DIRACGrid/DIRAC
-   :alt: Build Status
+.. image:: https://github.com/DIRACGrid/DIRAC/workflows/Basic%20tests/badge.svg?branch=rel-v7r0
```

Comment:
Should this be `master` instead?

Reply:
yes, use `master`

Reply:
The issue is that there are no workflows running on master as it's a copy of another branch (so the CI has already been ran). See: https://github.com/DIRACGrid/DIRAC/actions?query=workflow%3A%22Basic+tests%22+branch%3Amaster

I'll look into if I can find a work around...

Reply:
ouf... right. The workaround is to to use the latest tag that's not a "pre" tag...

Reply:
I have no idea how to encode that requirement in the URL. I think the best hope is to rerun the CI for the master branch when commits are reran.

Alternatively the read me could have one section for each supported version + integration?

Reply:
Then the only way is to modify the README of integration when `master` changes. I mean, you set right now `rel-v7r0` (because right now `master` == `rel-v7r0`, and then you update the instructions end of **https://dirac.readthedocs.io/en/latest/DeveloperGuide/ReleaseProcedure/index.html#propagating-patches**